habitat/plan.sh

```diff
@@ -0,0 +1,59 @@
+pkg_name=knife-ec-backup
+pkg_origin=chef
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_description="A knife plugin for backup/restore of Chef Server data."
+pkg_upstream_url="https://github.com/chef/knife-ec-backup"
+pkg_license=('Apache-2.0')
+pkg_bin_dirs=(bin)
+pkg_lib_dirs=(lib)
+pkg_svc_user=root
+pkg_svc_group=${pkg_svc_user}
+pkg_build_deps=(
+  core/gcc-libs
+  core/libffi
+  core/make
+  core/openssl
+)
+pkg_deps=(
+  core/bundler
+  core/coreutils
+  core/gcc
+  core/ruby
+)
+
+pkg_build_deps=(
+  core/git
+  core/make
+  core/postgresql-client
+)
+
+do_unpack() {
+  mkdir -p "${HAB_CACHE_SRC_PATH}/${pkg_dirname}"
+  cp -r ../* ${HAB_CACHE_SRC_PATH}/${pkg_dirname}/
+}
+
+
+do_build() {
+  pushd "${HAB_CACHE_SRC_PATH}/${pkg_dirname}"
+  bundle install --jobs 2 --retry 5 --path ./vendor/bundle --binstubs
+  popd
+}
+
+do_install() {
+  pushd "${HAB_CACHE_SRC_PATH}/${pkg_dirname}"
+  cp -R . "$pkg_prefix/"
+  fix_interpreter "$pkg_prefix/bin/knife" core/coreutils bin/env
```

Comment:
`fix_interpreter` doesn't need to be run in any specific path, I would recommend putting it out to avoid confusions.

Reply:
@afiune so just `fix_interpreter`?

Reply:
Yes, just `fix_interpreter`. You used it correctly but I was just being picky here and trying to separate the commands that really need to be run inside the directory you are switching to, `fix_interpreter` can be run outside sine you have to specify the path as a parameter.

Suggestion for this `do_install()` function:
```
do_install() {
  cp -R "${HAB_CACHE_SRC_PATH}/${pkg_dirname}/." "$pkg_prefix/"
  fix_interpreter "$pkg_prefix/bin/knife" core/coreutils bin/env
}
```
**NOTE**: You don't need to change directories to copy them and then, fix the interpreter of the knife binary.